src/main/java/no/nav/tag/finnkandidatapi/aktørregister/AktørRegisterClient.java

```diff
@@ -0,0 +1,81 @@
+package no.nav.tag.finnkandidatapi.aktørregister;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.tag.finnkandidatapi.kandidat.FinnKandidatException;
+import no.nav.tag.finnkandidatapi.tilgangskontroll.sts.STSClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Slf4j
+@Component
+public class AktørRegisterClient {
```

Comment:
Hva med en test for denne klassen? Det er jo en del logikk med f.eks. validering.